runtime/features/org.eclipse.core.runtime.feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <feature
       id="org.eclipse.core.runtime.feature"
       label="%featureName"
-      version="1.4.600.qualifier"
+      version="1.4.700.qualifier"
```

Comment:
why did the bot not increase the version of the feature?

Reply:
@HannesWell: Can you answer this question?

Reply:
I assume it's because the included `org.eclipse.equinox.core.feature`, which is maintained in the equinox repository was changed due to comparator-errors:
- https://github.com/eclipse-equinox/equinox/pull/707

And since the content changed, the feature needs a version bump, but it looks like this was already done and rebasing would let the change vanish:
- https://github.com/eclipse-platform/eclipse.platform/pull/1639

Reply:
Thanks @HannesWell.